src/liballoc/string.rs

```diff
@@ -429,7 +429,7 @@ impl String {
 
     /// Converts a vector of bytes to a `String`.
     ///
-    /// A string slice ([`&str`]) is made of bytes ([`u8`]), and a vector of bytes
+    /// A string slice ([`String`]) is made of bytes ([`u8`]), and a vector of bytes
```

Comment:
```suggestion
    /// A string ([`String`]) is made of bytes ([`u8`]), and a vector of bytes
```

Reply:
And, below

`The inverse of this method is [`as_bytes`].` -> `The inverse of this method is [`into_bytes`].`

Reply:
Made the changes